### DISCUSSION TITLE
打补丁的问题

### DISCUSSION THREAD

**Discussion by jzliming** [rime/home#1556] [Q&A]
![想打补丁的地方](https://github.com/rime/home/assets/52204784/c5f78217-18ac-4945-a799-8b5dd8554afa)
![{BB4F39A3-6674-4b0d-A590-D8CAE93C789E}](https://github.com/rime/home/assets/52204784/a15bed53-8929-4056-b7cc-e627f35ba94a)
build里的相应的文件也变了，为什么，没生效
patch:
  switches/@5/reset: 1
  popping/__append/@4/prefix: 2
这是打的方法，是我打的不对么，不懂。

**Comment by fxliang** [comment]
部署成功，说明配置文件组织librime已经解析正确生成部署配置。
但是这个似乎是声笔的专用的配置，librime自有的组件不会处理，大概你要找声笔仓库提issue。https://github.com/sbsrf/sbsrf/issues
